### PR TITLE
Implement the AngularMomentumTask in TSID component

### LINKED ISSUE #434: Implement angular momentum task in TSID
This issue is to keep in mind that I should implement the angular momentum task in TSID

### PR DESCRIPTION
**Pull request by GiulioRomualdi**
This closes #434

### CODE REVIEW

File: src/TSID/include/BipedalLocomotion/TSID/AngularMomentumTask.h [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+namespace BipedalLocomotion
+{
+namespace TSID
+{
+
+/**
+ * AngularMomentumTask is a concrete implementation of the TSIDLinearTask. Please use this element
+ * if you want to control the Centroidal angular momentum of the robot.
+ * The task represents the following equation
+ * The task represents the following equation
```
Comment (S-Dafarra) [inline src/TSID/include/BipedalLocomotion/TSID/AngularMomentumTask.h:33]: Duplicated

**Review by S-Dafarra — APPROVED** [review]

**Comment by S-Dafarra** [comment]
Btw, now the tests take more than an hour to complete. I guess it would be better to simplify them, avoiding a single test to run for more than 30sec for example

**Comment by GiulioRomualdi** [comment]
> Btw, now the tests take more than an hour to complete. I guess it would be better to simplify them, avoiding a single test to run for more than 30sec for example

I can open a PR just for that 😃

File: src/TSID/include/BipedalLocomotion/TSID/AngularMomentumTask.h (hunk 1 above)
Comment (GiulioRomualdi) [reply src/TSID/include/BipedalLocomotion/TSID/AngularMomentumTask.h]: Fixed

**Review by traversaro — APPROVED** [review]

**Pull request merged**